import 'dexie-observable': Cannot augment module 'Dexie' with value exports because it resolves to a non-module entity.

I'm trying to include dexie-observable, however the TS transpiler seems to have an issue with that:

```
ERROR in node_modules/dexie-observable/dist/dexie-observable.d.ts(13,15): error TS2649: Cannot augment module 'Dexie' with value exports because it resolves to a non-module entity.
node_modules/dexie-observable/dist/dexie-observable.d.ts(17,29): error TS2694: Namespace '"@paperwork/web/node_modules/dexie/dist/dexie".Dexie' has no exported member 'Observable'.
node_modules/dexie-observable/dist/dexie-observable.d.ts(34,40): error TS2694: Namespace '"@paperwork/web/node_modules/dexie/dist/dexie".Dexie' has no exported member 'Observable'.
node_modules/dexie-observable/dist/dexie-observable.d.ts(37,39): error TS2694: Namespace '"@paperwork/web/node_modules/dexie/dist/dexie".Dexie' has no exported member 'Observable'.
node_modules/dexie-observable/dist/dexie-observable.d.ts(46,60): error TS2694: Namespace '"@paperwork/web/node_modules/dexie/dist/dexie".Dexie' has no exported member 'Observable'.
node_modules/dexie-observable/dist/dexie-observable.d.ts(99,56): error TS2694: Namespace '"@paperwork/web/node_modules/dexie/dist/dexie".Dexie' has no exported member 'DexieEventSet'.
node_modules/dexie-observable/dist/dexie-observable.d.ts(120,22): error TS2339: Property 'Observable' does not exist on type 'DexieConstructor'.
src/app/app.component.ts(51,11): error TS2345: Argument of type '"changes"' is not assignable to parameter of type '"versionchange"'.
```

Also, this time it doesn't seem to be caching related. :)

--- a/addons/Dexie.Observable/package.json
+++ b/addons/Dexie.Observable/package.json
@@ -1,6 +1,6 @@
 {
   "name": "dexie-observable",
-  "version": "2.0.0-beta.9",
+  "version": "3.0.0-beta.9",
   "description": "Addon to Dexie that makes it possible to observe database changes no matter if they occur on other db instance or other window.",
   "main": "dist/dexie-observable.js",
   "module": "dist/dexie-observable.es.js",

--- a/addons/Dexie.Observable/package.json
+++ b/addons/Dexie.Observable/package.json
@@ -1,6 +1,6 @@
 {
   "name": "dexie-observable",
-  "version": "1.0.0-beta.8",
+  "version": "2.0.0-beta.9",
   "description": "Addon to Dexie that makes it possible to observe database changes no matter if they occur on other db instance or other window.",
   "main": "dist/dexie-observable.js",
   "module": "dist/dexie-observable.es.js",
@@ -76,10 +76,10 @@
   },
   "homepage": "http://dexie.org",
   "peerDependencies": {
-    "dexie": "^2.0.4"
+    "dexie": "^3.0.2"
   },
   "devDependencies": {
-    "dexie": "^2.0.4",
+    "dexie": "^3.0.2",
     "eslint": "^5.16.0",
     "just-build": "^0.9.19",
     "karma": "^4.1.0",

--- a/addons/Dexie.Observable/src/Dexie.Observable.d.ts
+++ b/addons/Dexie.Observable/src/Dexie.Observable.d.ts
@@ -1,4 +1,4 @@
-// Type definitions for dexie-observable v1.0.0-beta.8
+// Type definitions for dexie-observable v{version}
 // Project: https://github.com/dfahlander/Dexie.js/tree/master/addons/Dexie.Observable
 // Definitions by: David Fahlander <http://github.com/dfahlander>
 
@@ -58,6 +58,7 @@ declare module 'dexie' {
         // Placeholder where to access the SyncNode class constructor.
         // (makes it valid to do new db.observable.SyncNode())
         observable: {
+            version: string;
             SyncNode: SyncNodeConstructor;
             sendMessage(
                 type: string, // Don't use 'response' as it is used internally by the framework
@@ -98,9 +99,7 @@ declare module 'dexie' {
         Observable: {
             (db: Dexie) : void;
 
-            //
-            //
-            //
+            version: string;
             createUUID: () => string;
             on: ObservableEventSet;
             localStorageImpl: {

--- a/addons/Dexie.Observable/src/Dexie.Observable.d.ts
+++ b/addons/Dexie.Observable/src/Dexie.Observable.d.ts
@@ -1,9 +1,53 @@
-// Type definitions for dexie-observable v{version}
+// Type definitions for dexie-observable v1.0.0-beta.8
 // Project: https://github.com/dfahlander/Dexie.js/tree/master/addons/Dexie.Observable
 // Definitions by: David Fahlander <http://github.com/dfahlander>
 
-import Dexie from 'dexie';
+import Dexie, { DexieEventSet } from 'dexie';
 import { IDatabaseChange } from '../api';
+
+export interface SyncNodeConstructor {
+    new() : SyncNode;
+}
+
+//
+// Interfaces of Dexie.Observable
+//            
+
+
+/**
+ * A SyncNode represents a local database instance that subscribes
+ * to changes made on the database.
+ * SyncNodes are stored in the _syncNodes table.
+ * 
+ * Dexie.Syncable extends this interface and allows 'remote' nodes to be stored
+ * as well.
+ */
+export interface SyncNode {
+    id?: number,
+    myRevision: number,
+    type: 'local' | 'remote',
+    lastHeartBeat: number,
+    deleteTimeStamp: number, // In case lastHeartBeat is too old, a value of now + HIBERNATE_GRACE_PERIOD will be set here. If reached before node wakes up, node will be deleted.
+    isMaster: number // 1 if true. Not using Boolean because it's not possible to index Booleans.
+}
+
+export interface ObservableEventSet extends DexieEventSet {
+    (eventName: 'latestRevisionIncremented', subscriber: (dbName: string, latestRevision: number) => void): void;
+    (eventName: 'suicideNurseCall', subscriber: (dbName: string, nodeID: number) => void): void;
+    (eventName: 'intercomm', subscriber: (dbName: string) => void): void;
+    (eventName: 'beforeunload', subscriber: () => void): void;
+}
+
+// Object received by on('message') after sendMessage() or broadcastMessage()
+interface MessageEvent {
+    id: number;
+    type: string;
+    message: any;
+    destinationNode: number;
+    wantReply?: boolean;
+    resolve(result: any): void;
+    reject(error: any): void;
+}
 
 //
 // Extend Dexie interface
@@ -14,7 +58,7 @@ declare module 'dexie' {
         // Placeholder where to access the SyncNode class constructor.
         // (makes it valid to do new db.observable.SyncNode())
         observable: {
-            SyncNode: Dexie.Observable.SyncNodeConstructor;
+            SyncNode: SyncNodeConstructor;
             sendMessage(
                 type: string, // Don't use 'response' as it is used internally by the framework
                 message: any, // anything that can be saved by IndexedDB
@@ -31,34 +75,34 @@ declare module 'dexie' {
             ): void;
         }
 
-        readonly _localSyncNode: Dexie.Observable.SyncNode;
+        readonly _localSyncNode: SyncNode;
 
         _changes: Dexie.Table<IDatabaseChange & {rev: number}, number>;
-        _syncNodes: Dexie.Table<Dexie.Observable.SyncNode, number>;
+        _syncNodes: Dexie.Table<SyncNode, number>;
         _intercomm: Dexie.Table<any, number>;
     }
 
-    module Dexie {
-        // Extended events db.on('changes', subscriber), ...
-        interface DbEvents {
-            (eventName: 'changes', subscriber: (changes: IDatabaseChange[], partial: boolean)=>void): void;
-            (eventName: 'cleanup', subscriber: ()=>any): void;
-            (eventName: 'message', subscriber: (msg: Dexie.Observable.MessageEvent)=>any): void;
-        }
+    // Extended events db.on('changes', subscriber), ...
+    interface DbEvents {
+        (eventName: 'changes', subscriber: (changes: IDatabaseChange[], partial: boolean)=>void): void;
+        (eventName: 'cleanup', subscriber: ()=>any): void;
+        (eventName: 'message', subscriber: (msg: MessageEvent)=>any): void;
+    }
 
-        // Extended IndexSpec with uuid boolean for primary key.
-        interface IndexSpec {
-            uuid: boolean;
-        }
+    // Extended IndexSpec with uuid boolean for primary key.
+    interface IndexSpec {
+        uuid: boolean;
+    }
 
-        var Observable: {
+    interface DexieConstructor {
+        Observable: {
             (db: Dexie) : void;
 
             //
             //
             //
             createUUID: () => string;
-            on: Observable.ObservableEventSet;
+            on: ObservableEventSet;
             localStorageImpl: {
                 setItem(key: string, value: string): void,
                 getItem(key: string): string,
@@ -67,53 +111,6 @@ declare module 'dexie' {
             _onStorage: (event: StorageEvent) => void;
         }
 
-        //
-        // Define Dexie.Observable
-        //
-        module Observable {
-            //
-            // Interfaces of Dexie.Observable
-            //            
-
-            interface SyncNodeConstructor {
-                new() : SyncNode;
-            }
-
-            /**
-             * A SyncNode represents a local database instance that subscribes
-             * to changes made on the database.
-             * SyncNodes are stored in the _syncNodes table.
-             * 
-             * Dexie.Syncable extends this interface and allows 'remote' nodes to be stored
-             * as well.
-             */
-            interface SyncNode {
-                id?: number,
-                myRevision: number,
-                type: 'local' | 'remote',
-                lastHeartBeat: number,
-                deleteTimeStamp: number, // In case lastHeartBeat is too old, a value of now + HIBERNATE_GRACE_PERIOD will be set here. If reached before node wakes up, node will be deleted.
-                isMaster: number // 1 if true. Not using Boolean because it's not possible to index Booleans.
-            }
-
-            interface ObservableEventSet extends Dexie.DexieEventSet {
-                (eventName: 'latestRevisionIncremented', subscriber: (dbName: string, latestRevision: number) => void): void;
-                (eventName: 'suicideNurseCall', subscriber: (dbName: string, nodeID: number) => void): void;
-                (eventName: 'intercomm', subscriber: (dbName: string) => void): void;
-                (eventName: 'beforeunload', subscriber: () => void): void;
-            }
-
-            // Object received by on('message') after sendMessage() or broadcastMessage()
-            interface MessageEvent {
-                id: number;
-                type: string;
-                message: any;
-                destinationNode: number;
-                wantReply?: boolean;
-                resolve(result: any): void;
-                reject(error: any): void;
-            }
-        }
     }
 }
 

--- a/addons/Dexie.Observable/src/Dexie.Observable.js
+++ b/addons/Dexie.Observable/src/Dexie.Observable.js
@@ -55,11 +55,17 @@ var override = Dexie.override;
 var Promise = Dexie.Promise;
 var browserIsShuttingDown = false;
 
+/** Dexie addon for change tracking and real-time observation.
+ * 
+ * @param {Dexie} db 
+ */
 export default function Observable(db) {
-    /// <summary>
-    ///   Extension to Dexie providing Syncronization capabilities to Dexie.
-    /// </summary>
-    /// <param name="db" type="Dexie"></param>
+    if (!/^3\./.test(Dexie.version))
+        throw new Error(`Missing dexie version 3.x`);
+    if (db.observable) {
+        if (db.observable.version !== "{version}") throw new Error(`Mixed versions of dexie-observable`);
+        return; // Addon already active.
+    }
 
     var NODE_TIMEOUT = 20000, // 20 seconds before local db instances are timed out. This is so that old changes can be deleted when not needed and to garbage collect old _syncNodes objects.
         HIBERNATE_GRACE_PERIOD = 20000, // 20 seconds
@@ -99,7 +105,7 @@ export default function Observable(db) {
         }
     });
 
-    db.observable = {};
+    db.observable = {version: "{version}"};
     db.observable.SyncNode = SyncNode;
 
     const wakeupObservers = initWakeupObservers(db, Observable, localStorage);
@@ -488,6 +494,7 @@ export default function Observable(db) {
 // Static properties and methods
 // 
 
+Observable.version = "{version}";
 Observable.latestRevision = {}; // Latest revision PER DATABASE. Example: Observable.latestRevision.FriendsDB = 37;
 Observable.on = Dexie.Events(null, "latestRevisionIncremented", "suicideNurseCall", "intercomm", "beforeunload"); // fire(dbname, value);
 Observable.createUUID = createUUID;
@@ -511,6 +518,13 @@ if (global.addEventListener) {
     global.addEventListener("storage", Observable._onStorage);
     global.addEventListener("beforeunload", Observable._onBeforeUnload);
 }
-// Register addon:
-Dexie.Observable = Observable;
-Dexie.addons.push(Observable);
+
+if (Dexie.Observable) {
+    if (Dexie.Observable.version !== "{version}") {
+        throw new Error (`Mixed versions of dexie-observable`);
+    }
+} else {
+    // Register addon:
+    Dexie.Observable = Observable;
+    Dexie.addons.push(Observable);
+}

--- a/addons/Dexie.Observable/src/Dexie.Observable.js
+++ b/addons/Dexie.Observable/src/Dexie.Observable.js
@@ -59,7 +59,7 @@ var browserIsShuttingDown = false;
  * 
  * @param {Dexie} db 
  */
-export default function Observable(db) {
+function Observable(db) {
     if (!/^3\./.test(Dexie.version))
         throw new Error(`Missing dexie version 3.x`);
     if (db.observable) {
@@ -528,3 +528,5 @@ if (Dexie.Observable) {
     Dexie.Observable = Observable;
     Dexie.addons.push(Observable);
 }
+
+export default Dexie.Observable;

--- a/addons/Dexie.Observable/tools/build-configs/rollup.config.js
+++ b/addons/Dexie.Observable/tools/build-configs/rollup.config.js
@@ -9,14 +9,14 @@ export default {
   output: [{
     file: 'dist/dexie-observable.js',
     format: 'umd',
-    banner: readFileSync(path.resolve(__dirname, 'banner.txt'),
+    banner: readFileSync(path.resolve(__dirname, 'banner.txt')),
     globals: {dexie: "Dexie"},
     name: 'Dexie.Observable',
     sourcemap: true
   },{
     file: 'dist/dexie-observable.es.js',
     format: 'es',
-    banner: readFileSync(path.resolve(__dirname, 'banner.txt'),
+    banner: readFileSync(path.resolve(__dirname, 'banner.txt')),
     globals: {dexie: "Dexie"},
     name: 'Dexie.Observable',
     sourcemap: true

--- a/addons/Dexie.Observable/tools/build-configs/rollup.config.js
+++ b/addons/Dexie.Observable/tools/build-configs/rollup.config.js
@@ -9,18 +9,14 @@ export default {
   output: [{
     file: 'dist/dexie-observable.js',
     format: 'umd',
-    banner: readFileSync(path.resolve(__dirname, 'banner.txt'))+""
-        .replace(/{version}/g, version)
-        .replace(/{date}/g, new Date().toDateString()),
+    banner: readFileSync(path.resolve(__dirname, 'banner.txt'),
     globals: {dexie: "Dexie"},
     name: 'Dexie.Observable',
     sourcemap: true
   },{
     file: 'dist/dexie-observable.es.js',
     format: 'es',
-    banner: readFileSync(path.resolve(__dirname, 'banner.txt'))+""
-        .replace(/{version}/g, version)
-        .replace(/{date}/g, new Date().toDateString()),
+    banner: readFileSync(path.resolve(__dirname, 'banner.txt'),
     globals: {dexie: "Dexie"},
     name: 'Dexie.Observable',
     sourcemap: true

--- a/addons/Dexie.Syncable/package.json
+++ b/addons/Dexie.Syncable/package.json
@@ -1,6 +1,6 @@
 {
   "name": "dexie-syncable",
-  "version": "1.0.0-beta.8",
+  "version": "2.0.0-beta.9",
   "description": "Addon to Dexie that makes it possible to sync indexeDB with remote databases.",
   "main": "dist/dexie-syncable.js",
   "module": "dist/dexie-syncable.es.js",
@@ -70,12 +70,12 @@
   },
   "homepage": "http://dexie.org",
   "peerDependencies": {
-    "dexie": "^2.0.0-beta.7",
-    "dexie-observable": "^0.9.2 || ^1.0.0-beta"
+    "dexie": "^3.0.2",
+    "dexie-observable": "^2.0.0-beta.9"
   },
   "devDependencies": {
-    "dexie": "^2.0.4",
-    "dexie-observable": "^1.0.0-beta.5",
+    "dexie": "^3.0.2",
+    "dexie-observable": "^2.0.0-beta.9",
     "eslint": "^5.16.0",
     "just-build": "^0.9.19",
     "karma": "^4.1.0",

--- a/addons/Dexie.Syncable/package.json
+++ b/addons/Dexie.Syncable/package.json
@@ -1,6 +1,6 @@
 {
   "name": "dexie-syncable",
-  "version": "2.0.0-beta.9",
+  "version": "3.0.0-beta.9",
   "description": "Addon to Dexie that makes it possible to sync indexeDB with remote databases.",
   "main": "dist/dexie-syncable.js",
   "module": "dist/dexie-syncable.es.js",
@@ -71,11 +71,11 @@
   "homepage": "http://dexie.org",
   "peerDependencies": {
     "dexie": "^3.0.2",
-    "dexie-observable": "^2.0.0-beta.9"
+    "dexie-observable": "^3.0.0-beta.9"
   },
   "devDependencies": {
     "dexie": "^3.0.2",
-    "dexie-observable": "^2.0.0-beta.9",
+    "dexie-observable": "^3.0.0-beta.9",
     "eslint": "^5.16.0",
     "just-build": "^0.9.19",
     "karma": "^4.1.0",

--- a/addons/Dexie.Syncable/src/Dexie.Syncable.d.ts
+++ b/addons/Dexie.Syncable/src/Dexie.Syncable.d.ts
@@ -1,11 +1,16 @@
-// Type definitions for dexie-syncable v{version}
+// Type definitions for dexie-syncable v1.0.0-beta.8
 // Project: https://github.com/dfahlander/Dexie.js/tree/master/addons/Dexie.Syncable
 // Definitions by: David Fahlander <http://github.com/dfahlander>
 
-import Dexie from 'dexie';
+import Dexie, { DexieEventSet } from 'dexie';
 import 'dexie-observable';
 import { ISyncProtocol, SyncStatus } from '../api';
 import {IDatabaseChange} from 'dexie-observable/api';
+
+
+export interface SyncableEventSet extends DexieEventSet {
+    (eventName: 'statusChanged', subscriber: (status: number, url: string) => void): void;
+}
 
 //
 // Extend Dexie interface
@@ -47,7 +52,7 @@ declare module 'dexie' {
              * Syncable events. See docs at:
              * https://github.com/dfahlander/Dexie.js/wiki/db.syncable.on('statusChanged')
              */
-            on: Dexie.Syncable.SyncableEventSet;
+            on: SyncableEventSet;
         }
 
         /**
@@ -60,29 +65,8 @@ declare module 'dexie' {
         _uncommittedChanges: Dexie.Table<IDatabaseChange & {id: number, node: number}, number>;
     }
 
-    module Dexie {
-
-        // Extend SyncNode interface from Dexie.Observable to
-        // allow storing remote nodes in table _syncNodes.
-        module Observable {
-            interface SyncNode {
-                url: string, // Only applicable for "remote" nodes. Only used in Dexie.Syncable.
-                syncProtocol: string, // Tells which implementation of ISyncProtocol to use for remote syncing.
-                syncContext: any,
-                syncOptions: any,
-                status: number,
-                appliedRemoteRevision: any,
-                remoteBaseRevisions: { local: number, remote: any }[],
-                dbUploadState: {
-                    tablesToUpload: string[],
-                    currentTable: string,
-                    currentKey: any,
-                    localBaseRevision: number
-                }
-            }
-        }
-
-        var Syncable : {
+    interface DexieConstructor {
+        Syncable: {
             (db: Dexie) : void;
             /**
              * See documentation at:
@@ -93,11 +77,28 @@ declare module 'dexie' {
             /** Translates a sync status number into a string "ERROR_WILL_RETRY", "ERROR", etc */
             StatusTexts: {[syncStatus:number]: string};
         }
+    }
+}
 
-        module Syncable {
-            interface SyncableEventSet extends DexieEventSet {
-                (eventName: 'statusChanged', subscriber: (status: number, url: string) => void): void;
-            }
+//
+// Extend dexie-observable interfaces
+//
+declare module "dexie-observable" {
+    // Extend SyncNode interface from Dexie.Observable to
+    // allow storing remote nodes in table _syncNodes.
+    interface SyncNode {
+        url: string, // Only applicable for "remote" nodes. Only used in Dexie.Syncable.
+        syncProtocol: string, // Tells which implementation of ISyncProtocol to use for remote syncing.
+        syncContext: any,
+        syncOptions: any,
+        status: number,
+        appliedRemoteRevision: any,
+        remoteBaseRevisions: { local: number, remote: any }[],
+        dbUploadState: {
+            tablesToUpload: string[],
+            currentTable: string,
+            currentKey: any,
+            localBaseRevision: number
         }
     }
 }

--- a/addons/Dexie.Syncable/src/Dexie.Syncable.d.ts
+++ b/addons/Dexie.Syncable/src/Dexie.Syncable.d.ts
@@ -1,4 +1,4 @@
-// Type definitions for dexie-syncable v1.0.0-beta.8
+// Type definitions for dexie-syncable v{version}
 // Project: https://github.com/dfahlander/Dexie.js/tree/master/addons/Dexie.Syncable
 // Definitions by: David Fahlander <http://github.com/dfahlander>
 
@@ -18,6 +18,7 @@ export interface SyncableEventSet extends DexieEventSet {
 declare module 'dexie' {
     interface Dexie {
         syncable: {
+            version: string;
             /**
              * Connect to given URL using given protocol and options. See documentation at:
              * https://github.com/dfahlander/Dexie.js/wiki/db.syncable.connect()
@@ -68,9 +69,12 @@ declare module 'dexie' {
     interface DexieConstructor {
         Syncable: {
             (db: Dexie) : void;
+
+            version: string;
+
             /**
              * See documentation at:
-             * https://github.com/dfahlander/Dexie.js/wiki/Dexie.Syncable.registerSyncProtocol()
+             * https://dexie.org/docs/Syncable/Dexie.Syncable.registerSyncProtocol()
              */
             registerSyncProtocol: (name: string, prototocolInstance: ISyncProtocol) => void;
 

--- a/addons/Dexie.Syncable/src/Dexie.Syncable.js
+++ b/addons/Dexie.Syncable/src/Dexie.Syncable.js
@@ -32,8 +32,19 @@ var override = Dexie.override,
     Promise = Dexie.Promise,
     Observable = Dexie.Observable;
 
+/** Dexie addon for remote database sync.
+ * 
+ * @param {Dexie} db 
+ */
 export default function Syncable (db) {
-    /// <param name="db" type="Dexie"></param>
+    if (!/^3\./.test(Dexie.version))
+        throw new Error(`Missing dexie version 3.x`);
+    if (!db.observable || (db.observable.version !== "{version}" && !/^3\./.test(db.observable.version)))
+        throw new Error(`Missing dexie-observable version 3.x`);
+    if (db.syncable) {
+        if (db.syncable.version !== "{version}") throw new Error(`Mixed versions of dexie-syncable`);
+        return; // Addon already active.
+    }
 
     var activePeers = [];
 
@@ -114,7 +125,9 @@ export default function Syncable (db) {
         }
     }, true); // True means the ready event will survive a db reopen - db.close()/db.open()
 
-    db.syncable = {};
+    db.syncable = {
+        version: "{version}"
+    };
 
     db.syncable.getStatus = function(url, cb) {
         if (db.isOpen()) {
@@ -236,6 +249,8 @@ export default function Syncable (db) {
      });
 }
 
+Syncable.version = "{version}";
+
 Syncable.Statuses = Statuses;
 
 Syncable.StatusTexts = StatusTexts;
@@ -262,6 +277,12 @@ Syncable.registerSyncProtocol = function(name, protocolInstance) {
     Syncable.registeredProtocols[name] = protocolInstance;
 };
 
-// Register addon in Dexie:
-Dexie.Syncable = Syncable;
-Dexie.addons.push(Syncable);
+if (Dexie.Syncable) {
+    if (Dexie.Syncable.version !== "{version}") {
+        throw new Error (`Mixed versions of dexie-syncable`);
+    }
+} else {
+    // Register addon in Dexie:
+    Dexie.Syncable = Syncable;
+    Dexie.addons.push(Syncable);
+}

--- a/addons/Dexie.Syncable/src/Dexie.Syncable.js
+++ b/addons/Dexie.Syncable/src/Dexie.Syncable.js
@@ -36,7 +36,7 @@ var override = Dexie.override,
  * 
  * @param {Dexie} db 
  */
-export default function Syncable (db) {
+function Syncable (db) {
     if (!/^3\./.test(Dexie.version))
         throw new Error(`Missing dexie version 3.x`);
     if (!db.observable || (db.observable.version !== "{version}" && !/^3\./.test(db.observable.version)))
@@ -286,3 +286,5 @@ if (Dexie.Syncable) {
     Dexie.Syncable = Syncable;
     Dexie.addons.push(Syncable);
 }
+
+export default Dexie.Syncable;

--- a/addons/Dexie.Syncable/tools/build-configs/rollup.config.js
+++ b/addons/Dexie.Syncable/tools/build-configs/rollup.config.js
@@ -14,9 +14,7 @@ export default {
     format: 'es'
   }],
   sourceMap: true,
-  banner: readFileSync(path.resolve(__dirname, 'banner.txt'))+""
-    .replace(/{version}/g, version)
-    .replace(/{date}/g, new Date().toDateString()),
+  banner: readFileSync(path.resolve(__dirname, 'banner.txt')),
   moduleName: 'Dexie.Syncable',
   globals: {dexie: "Dexie", "dexie-observable": "Dexie.Observable"},
   external: ['dexie', 'dexie-observable'],

--- a/addons/dexie-export-import/tools/build-configs/rollup.config.js
+++ b/addons/dexie-export-import/tools/build-configs/rollup.config.js
@@ -16,7 +16,7 @@ export default {
   output: [{
     file: 'dist/dexie-export-import.js',
     format: 'umd',
-    banner: readFileSync(path.resolve(__dirname, 'banner.txt'))+""
+    banner: readFileSync(path.resolve(__dirname, 'banner.txt'), "utf-8")
         .replace(/{version}/g, version)
         .replace(/{date}/g, new Date().toDateString()),
     globals: {dexie: "Dexie"},
@@ -26,7 +26,7 @@ export default {
   },{
     file: 'dist/dexie-export-import.mjs',
     format: 'es',
-    banner: readFileSync(path.resolve(__dirname, 'banner.txt'))+""
+    banner: readFileSync(path.resolve(__dirname, 'banner.txt'), "utf-8")
         .replace(/{version}/g, version)
         .replace(/{date}/g, new Date().toDateString()),
     sourcemap: true

